bfe.test.js

```diff
@@ -19,35 +19,39 @@ tape('bfe', function (t) {
     'format.code = Array index in bfe.json'
   )
 
-  const sigiledTypes = bfeTypes.reduce((acc, type) => {
-    if (type.sigil) return [...acc, type]
-    return acc
-  }, [])
-  t.equal(
-    sigiledTypes.length,
-    new Set(sigiledTypes.map((type) => type.sigil)).size, // unique sigil
-    'each sigil is unique to a type'
-  )
-  t.equal(sigiledTypes.length, 3, 'there are only 3 sigils')
-
-  sigiledTypes.forEach(type => {
-    const typeSuffixes = type.formats.reduce((acc, format) => {
-      if (format.suffix && format.suffix.length) acc.add(format.suffix)
-      return acc
-    }, new Set())
-
-    t.equal(
-      typeSuffixes.size,
-      type.formats.length,
-      `each type/format with sigil ${type.sigil} has unique suffix` // unique to that type
-    )
-  })
-
-  const sigillessSuffixFormats = bfeTypes.reduce((acc, type) => {
-    if (type.sigil) return acc
-
-    return [...acc, ...type.formats.filter((format) => format.suffix)]
-  }, [])
+  const sigils = new Set()
+  loopOverTypes: for (const type of bfeTypes) {
```

Comment:
This is new syntax to me!

Reply:
Maybe the first or second time ever that I find a legit use case for it :)

Reply:
What's it called so I can learn what it is

Reply:
Loop labels? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue